scripts/multisrc/lightnovelwp/sources.json

```diff
@@ -176,5 +176,32 @@
       "lang": "Portuguese",
       "reverseChapters": true
     }
+  },
+  {
+    "id": "ellotl",
+    "sourceSite": "https://ellotl.com/",
+    "sourceName": "ElloTL",
+    "options": {
+      "lang": "English",
+      "reverseChapters": true,
+      "versionIncrements": 1
+    }
+  },
+  {
+    "id": "ippotranslations",
+    "sourceSite": "https://ippotranslations.com/",
+    "sourceName": "Ippotranslations",
+    "options": {
+      "lang": "English"
+    }
+  },
+  {
+    "id": "cpunovel",
+    "sourceSite": "https://cpunovel.com/",
+    "sourceName": "CPUnovel",
+    "options": {
+      "lang": "English",
+      "reverseChapters": true
+    }
```

Comment:
Cpunovel isn't a new site it's juste the new name of daotranslate

Reply:
they have different pages of `popular novels`, `recent releases`,
I think it's a new site.